CONTRIBUTING.md

```diff
@@ -102,6 +102,8 @@ Most operating systems provide all the needed tools (including Windows, Linux an
   - [Docker](https://docs.docker.com/get-docker/)
     - It is not mandatory because you can easily locally run tests against SQLite without it.
     - It is practically mandatory if you want to locally run tests against any other database engine (MySQL, MariaDB, Postgres,Db2 and MSSQL), unless you happen to have the engine installed and is willing to make some manual configuration.
+    - [Docker Compose Plugin](https://docs.docker.com/compose/install/)
+      - It is mandatory if you are wanting to locally run tests using the docker containers.
```

Comment:
```suggestion
  - [Docker](https://docs.docker.com/get-docker/) and [Docker Compose Plugin](https://docs.docker.com/compose/install/)
    - It is not mandatory because you can easily locally run tests against SQLite without it.
    - It is practically mandatory if you want to locally run tests against any other database engine (MySQL, MariaDB, Postgres,Db2 and MSSQL), unless you happen to have the engine installed and is willing to make some manual configuration.
```

This should already be enough, right?